ansible/roles/postgresql/templates/backup/plain/create_postgres_dump.sh.j2

```diff
@@ -1,12 +1,20 @@
 #!/bin/bash
 BACKUP_TYPE=$1
 DAYS_TO_RETAIN_BACKUPS=$2
+ENV="{{ deploy_env }}"
 
 TODAY=$(date +"%Y_%m_%d")
-DIRECTORY="{{ postgresql_backup_dir }}/postgres_${BACKUP_TYPE}_${TODAY}"
+DIRECTORY="{{ postgresql_backup_dir }}/postgres_${ENV}_${BACKUP_TYPE}_${TODAY}"
+
+{% if postgresql_backup_master|default(false) and hot_standby_master|default(false) %}
```

Comment:
Shouldn't this be `if not postgresql_backup_master`

Reply:
probable a bad var name, I meant backup_from_master

Reply:
The name is fine and that's what I assumed it meant but if we're backing up from the master why do we need to supply the host option (`-h`)?

Reply:
we do, line 11, pg_basebackup -h $MASTER -Ft -D "${DIRECTORY}" -Z9 -Xf

Reply:
offlined, I didn't realize the script was running on the standby node